packages/shorebird_cli/lib/src/shorebird_env.dart

```diff
@@ -29,6 +31,11 @@ class ShorebirdEnv {
 
   final String? _flutterRevisionOverride;
 
+  /// The application config directory for the Shorebird CLI.
+  Directory get configDirectory {
+    return Directory(applicationConfigHome(executableName));
+  }
```

Comment:
Ultra-nit: this could be a fat arrow

Reply:
I changed it to block because it wraps weirdly 😛